stored_requests/cache/cacher/cacher.go

```diff
@@ -0,0 +1,74 @@
+package cacher
+
+import (
+	"errors"
+)
+
+// Cacher is the cacher interface for the db_fetchers
+type Cacher interface {
+	// Configure opens a new connection to the cache database.
+	// This should be called once
+	Configure(settings *Settings) error
+
+	// Name of the cache client
+	Name() string
+
+	// Ping cache database
+	Ping() error
+
+	// Close connection to cache database
+	Close()
+
+	// Get a string based off of a key
+	Get(key string) (string, error)
+
+	// Set/Update a value with TTL in seconds
+	Set(key, value string, ttl uint) error
```

Comment:
One more I forgot... I think these Get and Set methods should take in a [Context](https://golang.org/pkg/context/). I don't see much support for them in the redis library you're using... but they're a pretty common Go convention, so we should probably have them just to future-proof the interface

Reply:
Most of the cache drivers that I've seen in Go don't use context but I'm happy to add them. 

References: 
- https://github.com/go-redis/redis
- https://github.com/aerospike/aerospike-client-go
- https://github.com/boltdb/bolt (potential alternative to in-memory)

Reply:
huh.... that... surprises me, but ok. I was working off of the conclusion of [this blog post](https://blog.golang.org/context):

> At Google, we require that Go programmers pass a Context parameter as the first argument to every function on the call path between incoming and outgoing requests.

But... that was over 3 years ago, so if libraries haven't fallen into line yet, they probably won't any time soon.

I have very little Go experience... so use your judgement here ^^. I'm cool with it either way.

Reply:
They could be doing it wrong. I'm happy to add it and that way if we need context later we won't need to change the interface.